config/routes.rb

```diff
@@ -12,10 +14,13 @@
   delete 'merchant/logout', to: 'm_sessions#destroy'
 
   post 'customer/login', to: 'c_sessions#create'
+  get 'product/new', to: 'products#new'
+  post '/products', to: 'products#create'
+
 
 resources :customers, only: [:new, :create]
 resources :c_sessions, only: [:new, :create]
 resources :m_sessions, only: [:new, :create]
 resources :merchants, only: [:new, :create, :index]
-
+resources :products, only: [:new, :create, :index]
```

Comment:
Duplikacja linijek na górze, resources zastępują nam get 'product/new', to: 'products#new' itd.